src/views/Pools/components/PoolCard.tsx

```diff
@@ -213,50 +80,12 @@ const PoolCard: React.FC<HarvestProps> = ({ pool }) => {
         endBlock={endBlock}
         isFinished={isFinished}
         poolCategory={poolCategory}
-        tokenName={earningToken.symbol}
+        tokenName={stakingToken.symbol}
```

Comment:
Why did you do this?

Reply:
```suggestion
        earningTokenName={earningToken.symbol}
        stakingTokenName={stakingToken.symbol}
```

Reply:
> Why did you do this?

There was a comment says it should be stakingToken